app/src/main/java/org/p2p/wallet/striga/signup/validation/PhoneNumberInputValidator.kt

```diff
@@ -1,18 +1,18 @@
 package org.p2p.wallet.striga.signup.validation
 
-import java.util.regex.Pattern
 import org.p2p.core.common.TextContainer
 import org.p2p.wallet.R
+import org.p2p.wallet.auth.repository.CountryCodeLocalRepository
 
-private val PATTERN = Pattern.compile(
-    "^\\+[0-9]{10,15}\$"
-)
-
-class PhoneNumberInputValidator : InputValidator {
+class PhoneNumberInputValidator(
```

Comment:
Please, add comment, what is the format of each value should be:
- is `regionCode` a phone country code (digits) or region itself ISO alpha2/3 code?
- if `regionCode` is phone country code, should it include plus sign?
- should `phoneNumber` be cleared of spaces?